src/Middleware/Diagnostics/src/StatusCodePage/StatusCodePagesExtensions.cs

```diff
@@ -174,7 +174,7 @@ public static IApplicationBuilder UseStatusCodePages(this IApplicationBuilder ap
                 throw new ArgumentNullException(nameof(app));
             }
 
-            return app.UseStatusCodePages(async context =>
+            Func<StatusCodeContext, Task> handler = async context =>
```

Comment:
```suggestion
            var handler = async context =>
```

Reply:
Pretty sure I tried this and it can't figure out what the type is supposed to be.

Reply:
What about?

```suggestion
            var handler = async (StatusCodeContext context) =>
```

Reply:
Oh yeah, you need parameters 😄